### DISCUSSION TITLE
how encode decode like with python3  struct.pack struct.unpack in yaml

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/nuclei#2270] [Ideas]
@ehsandeep 
how encode decode int to little-endian big-endian uint16 uint32

more see
https://docs.python.org/3/library/struct.html

thanks

**Comment by ehsandeep** [comment]
@hktalent this is being worked on here - https://github.com/projectdiscovery/dsl/pull/16

**Comment by hktalent** [comment]
@ehsandeep That's great. Thank you very much
